### PR TITLE
Amercan English: s -> z

### PR DESCRIPTION
**Pull request by allymparker**
Use American English as per  https://info.red-gate.com/display/PD/American+English

### CODE REVIEW

File: Source/ULibs.TinyJsonDeser/RELEASENOTES.md [hunk 1]
```diff
@@ -22,4 +22,4 @@
 
 ### Features
 
-- A lightweight thread-safe JsonDeserializer class that can deserialise json text to .NET types. It does not support binding to specific .NET types. Rather, objects are deserialised to instances of `IDictionary<string, object>`, arrays are deserialised to instances of `object[]`, and literals are deserialised to either a `bool`, `double`, `string` or `null`.
+- A lightweight thread-safe JsonDeserializer class that can deserialize json text to .NET types. It does not support binding to specific .NET types. Rather, objects are deserialized to instances of `IDictionary<string, object>`, arrays are deserialized to instances of `object[]`, and literals are deserialized to either a `bool`, `double`, `string` or `null`.
```
Comment (ChrisLambrou) [inline Source/ULibs.TinyJsonDeser/RELEASENOTES.md:25]: You might want to add a version bump with a Fix entry describing the change. It's trivial, but this is public, so... :)

Comment (ChrisLambrou) [reply Source/ULibs.TinyJsonDeser/RELEASENOTES.md:25]: Same goes for the other release notes in this PR. If you don't bump the version, the change won't be released.

**Comment by allymparker** [comment]
Thanks for the review - I think that's done (the version comes from the release notes right?) :)

**Comment by ChrisLambrou** [comment]
👍 :shipit:

**Comment by ChrisLambrou** [comment]
> Thanks for the review - I think that's done (the version comes from the release notes right?) :)

Yep!

**Pull request merged**